modules/swagger-annotations/src/main/java/io/swagger/oas/annotations/responses/ApiResponse.java

```diff
@@ -26,35 +26,31 @@
 import java.lang.annotation.Target;
 
 /**
- *
- *
- *
+ * Describes a single response from an API Operation, including design-time, static links to operations based on the response.
  **/
-
-
 @Target({ ElementType.METHOD })
 @Retention(RetentionPolicy.RUNTIME)
 @Inherited
 public @interface ApiResponse {
     /**
-     *
+     *A short description of the response. 
      **/
     String description() default "";
 
     /**
-     * the HTTP response code, or default, for the supplied response
+     * The HTTP response code, or 'default', for the supplied response.  May only have 1 default entry.  
      **/
-    String responseCode() default "";
+    String responseCode() default "default";
```

Comment:
in this particular case, the default responseCode is named "default" as per the spec, so I thought it would be good to have that as the default.